packages/docs/src/styles/_sections.scss

```diff
@@ -69,6 +69,7 @@
 }
 
 #{section-name("Labels")},
+#{section-name("Helper Text")},
```

Comment:
on second thought, im going to stick with `line-height` since that's what we did previously for simple labels too - if we want to change it, we should do so together in a later PR